### DISCUSSION TITLE
Is this project discountinued?

### DISCUSSION THREAD

**Discussion by peaklabs-dev** [upptime/upptime#840] [General] [answered]
There were no updates since 2020, is this project no longer under developement?

**Comment by AnandChowdhary** [comment, marked as answer]
We released something 3 days ago (https://github.com/upptime/uptime-monitor/releases), why do you think there were no updates since 2020?

**Comment by peaklabs-dev** [comment]
Because this is displayed on the main page of uptime:
![image](https://github.com/upptime/upptime/assets/122374094/b8d08698-4054-4ce0-9514-9cec748d6e13)

[a comment by an automated account is not shown]
